### PR TITLE
Adds stabilization cores + the compressor for them

### PR DESCRIPTION
**Pull request by MLGTASTICa**
## About The Pull Request
Adds stabilization core and the compressor
The compressor is a 2 part machine that  creates stabilization cores using all the available power in the grid its connected to
the quality of the core gets upgraded by the amount of  extra power in the grid , ontop of the needed minimum (36 MW)
the stabilizer heals the SM upon being smashed into it by hand. or thrown at it. the amount healed is 200 by default and its multiplied by its quality
A stabilizer core costs 4 sheets of Mettalic hydrogen , 30 plasma and 30 diamond , and is worth 10000 credits , the price is also multiplied by  its quality,  so go big and you get extra bucks
If our powernet system didn't suck total ass , i would've made this drain all the available power , but it indeed does suck ass and i can't be bothered to make a work around.
Credit to CeUvi for helping adjust the stabilization core sprite.
Credit to InfraRedBaron for making the stabilization core super and stabilization core good sprite
## Why It's Good For The Game
Technomancer content?
image
image

## Changelog
:cl:
add: Stabilization core , Nano compressor , a 2 part machine
add: New map for the nano compressor
/:cl:

### CODE REVIEW

**Comment by FortuneMaster** [comment]
without the power to do so, I veto this PR. There's no point in making it drain the power from the grid. There's no reason to require mettalic hydrogen since it's leftover material that should be deleted and you are cockblocking it in general since no one will be able to make it. Cut the damn price of plasma in 3 and diamond in 6. This will be already high enough for it.

**Comment by SirRichardFrancis** [comment]
I very much like that you work on techno content, and stabilization cores is very neat, but somewhat complex thing.
If anything, it should block any and all magic(aka bluespace) fuckery.
I have no idea why Reere wanted cores to _heal_ supermatter instead of fucking damaging it or preventing from functioning, but in any case Reere is not here anymore and this thing will be nothing more than site of pilgrimage for the guild, in their pursuit of ruining economy.
Also it looks kind of... magical. Which is actually good for part of the stabilization core, but not the structure itself.

TL;DR
It should interact with bluespace entities and events, healing SM is pretty useless imo.

**Comment by MLGTASTICa** [comment]
> I very much like that you work on techno content, and stabilization cores is very neat, but somewhat complex thing. If anything, it should block any and all magic(aka bluespace) fuckery. I have no idea why Reere wanted cores to _heal_ supermatter instead of fucking damaging it or preventing from functioning, but in any case Reere is not here anymore and this thing will be nothing more than site of pilgrimage for the guild, in their pursuit of ruining economy. Also it looks kind of... magical. Which is actually good for part of the stabilization core, but not the structure itself.
> 
> TL;DR It should interact with bluespace entities and events, healing SM is pretty useless imo.

I would add more content onto it but you know it takes time, + i want to get it done for this week.

**Comment by ValoTheValo** [comment]
Finally. Techno content. After 7000 years
